app/config/variables.php

```diff
@@ -615,6 +614,15 @@
                 'question' => '',
                 'filter' => ''
             ],
+            [
+                'name' => '_APP_EXECUTOR_VERSION',
+                'description' => 'Executor version',
+                'introduction' => '0.14.0',
+                'default' => 'v1',
```

Comment:
```suggestion
                'default' => 'v2',
```
In 0.14 we will use V2, right?

Reply:
There was never V1, so probably we will use V1 still. As executor is still part of Appwrite, we don't have versioning here yet.